app/mailers/notify_mailer.rb

```diff
@@ -112,10 +112,10 @@ def export_email(user, attachment)
     mail(to: @user.email, subject: "The export of your content is ready")
   end
 
-  def tag_moderator_confirmation_email(user, tag_name)
-    @tag_name = tag_name
+  def tag_moderator_confirmation_email(user, tag)
+    @tag = tag
```

Comment:
Amended the passing of parameters in this function from being @tag_name to @tag (the active record model) instead so that we can pass this model to tag_url in the tag_moderator_confirmation_email  view.